tests/attacks/test_threshold_attack.py

```diff
@@ -55,8 +55,8 @@ def setUpClass(cls):
         super().setUpClass()
 
         cls.n_test = 2
-        cls.x_test_mnist = cls.x_test_mnist[0 : cls.n_test]
-        cls.y_test_mnist = cls.y_test_mnist[0 : cls.n_test]
+        cls.x_test_mnist = cls.x_test_mnist[0: cls.n_test]
+        cls.y_test_mnist = cls.y_test_mnist[0: cls.n_test]
```

Comment:
Please revert these changes because they are not following Black style formatting.
```suggestion
        cls.x_test_mnist = cls.x_test_mnist[0 : cls.n_test]
        cls.y_test_mnist = cls.y_test_mnist[0 : cls.n_test]
```